distributor-standalone/deeplinks.md

```diff
@@ -21,6 +21,8 @@ Parameters can be combined and some have no visible effect unless they are used
 | mewsVoucherCode | a voucher code | `E1A71167851A30043B12` |
 | mewsRoute | [mewsRoute](./deeplinks.md#parameter-mewsroute) | `rooms` for rooms step |
 | mewsRoom | opens with specified room selected \(`aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee`\) | `da394bbb-9685-4bb8-9547-ab7300915967` |
+| mewsAdultCount  | preselect room occupancy with initial adult count. [More details](./deeplinks.md#open-with-specific-initial-occupancy) | `3`                                    |
+| mewsChildCount  | preselect room occupancy with initial child count. [More details](./deeplinks.md#open-with-specific-initial-occupancy) | `1`                                    |
```

Comment:
We should use the form as for other things in the table.

And what if we'd use a copy from our Mews guide: https://help.mews.com/s/article/set-up-your-mews-booking-engine?language=en_US
 
```suggestion
| mewsAdultCount  | number of adults that should be selected by default | `3`                                    |
| mewsChildCount  | number of children that should be selected by default | `1`                                    |
```

To make it slightly less technical sounding.

Does it make sound good to you?

Reply:
Yup, less technical sounding sounds good 👍